src/Symfony/Component/Translation/Translator.php

```diff
@@ -345,22 +345,18 @@ public function dumpCatalogue($locale, ConfigCacheInterface $cache)
         $this->initializeCatalogue($locale);
         $fallbackContent = $this->getFallbackContent($this->catalogues[$locale]);
 
-        $content = sprintf(<<<EOF
+        $content = $this->getCataloguePhp($locale, $this->catalogues[$locale]->all(), '');
```

Comment:
please use a different variable name for this too, to avoid making ``$content`` change its meaning between lines